### PR TITLE
Make PDB writer write correct CONECT records for multiple molecules

### PR DESCRIPTION
**Pull request by pckroon**

### CODE REVIEW

File: vermouth/pdb/pdb.py [hunk 1]
```diff
@@ -88,11 +88,11 @@ def keyfunc(graph, node_idx):
     # molecules in a system. Probably not a good idea
     nodeidx2atomid = {}
     atomid = 1
-    for molecule in system.molecules:
+    for idx, molecule in enumerate(system.molecules):
```
Comment (jbarnoud) [inline vermouth/pdb/pdb.py]: `idx` could be `m_idx` for consistency with the loop bellow.

**Comment by jbarnoud** [comment]
Could you add a test? It does not have to be anything complicated: just a couple of molecules with 3 or 4 node each, and making sure that the written record contains the expected indices.

**Comment by pckroon** [comment]
I hate it when you keep me to my own standards ;)

**Comment by jbarnoud** [comment]
I merge as soon as the tests are passing.

**Pull request merged**